src/main/java/ChatReques11.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques11 {
+    private String model;
+    private List<?> messages;
```

Comment:
Using wildcard type List<?> is not recommended, specify the type for better type safety (Severity: Minor)